meson.build

```diff
@@ -8,7 +8,7 @@
 project(
     'libnvme', ['c', 'cpp'],
     meson_version: '>= 0.47.0',
-    version: '1.0',
+    version: '1.0rc5',
```

Comment:
I was following what systemd is doing here and there were fallouts with a postfix by some tooling (sorry can't remember but I think we have some discussion on this in the 'issues'.